src/ets/scala/io/iohk/ethereum/ets/blockchain/BlockchainSuite.scala

```diff
@@ -16,8 +16,8 @@ object BlockchainSuite {
 
 class BlockchainSuite extends FreeSpec with Matchers with BeforeAndAfterAll with Logger {
 
-  val unsupportedNetworks = Set("Byzantium","Constantinople", "EIP158ToByzantiumAt5")
-  val supportedNetworks = Set("EIP150", "Frontier", "FrontierToHomesteadAt5", "Homestead", "HomesteadToEIP150At5", "HomesteadToDaoAt5", "EIP158")
+  val unsupportedNetworks = Set("Constantinople", "EIP158ToByzantiumAt5")
+  val supportedNetworks = Set("EIP150", "Frontier", "FrontierToHomesteadAt5", "Homestead", "HomesteadToEIP150At5", "HomesteadToDaoAt5", "EIP158", "Byzantium")
```

Comment:
To enable ets tests you would also need to add a `byzantium` config to file `BlockchainTestConfig.scala` (example: https://github.com/input-output-hk/mantis/blob/phase/release2_0/src/ets/scala/io/iohk/ethereum/ets/blockchain/BlockchainTestConfig.scala#L187)